openhands/core/logger.py

```diff
@@ -18,7 +18,7 @@
 
 # Structured logs with JSON, disabled by default
 LOG_JSON = os.getenv('LOG_JSON', 'False').lower() in ['true', '1', 'yes']
-LOG_JSON_LEVEL_KEY = os.getenv('LOG_JSON_LEVEL_KEY', 'level')
+LOG_JSON_LEVEL_KEY = os.getenv('LOG_JSON_LEVEL_KEY', 'severity')
```

Comment:
It won't hurt us to change this default because we override anyway but why is it changing here?

Reply:
The agent said that `test_logging.py` was failing, but I'll try reverting it.

Reply:
Yeah probably because it changed the tests